docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM centos:centos6
```

Comment:
You can only tell it to run CentOS 6 (latest branch) or CentOS 6.6 specifically 8https://registry.hub.docker.com/_/centos/).